subprojects/gradle/room-config/src/main/kotlin/com/avito/android/plugin/incremental/RoomIncrementalKaptSupportChecker.kt

```diff
@@ -0,0 +1,53 @@
+package com.avito.android.plugin.incremental
+
+import org.gradle.api.Project
+import org.jetbrains.kotlin.gradle.internal.KaptGenerateStubsTask
+import java.net.URLClassLoader
+import javax.annotation.processing.AbstractProcessor
+
+internal class RoomIncrementalKaptSupportChecker(
```

Comment:
Could we create a unit test for it?

Reply:
Not actual. Move this check from this PR to another task.